pages/components/card.html

```diff
@@ -106,8 +106,204 @@ <h1 clss="page-title">card</h1>
 
 
   <main class="main-content">
-
-    <iframe src="/codeblock/x.txt" frameborder="0" class="codeblock"></iframe>
+    <section class="cards">
+      <div class="card-container card-style-1">
+        <section class="card-primary-content">
+          <figure class="media">
+            <img src="https://picsum.photos/400/200" alt="img">
+            <figcaption>This is img caption</figcaption>
+          </figure>
+          <header class="card-header">
+            <h2>Title - Basic</h2>
+          </header>
+          <p class="card-body">
+            This hold the summary/ description
+          </p>
+        </section>
+        <section class="actions card-actions">
+          <section class="action-buttons">
+            <button class="btn bg-primary"><a href="/">Action 1</a></button>
+            <button class="btn outline-primary"><a href="/">Action 2</a></button>
+          </section>
+          <section class="action-icons">
+            <button class="btn btn-icon">
+              <span><img src="/assets/like.svg" alt="like" class="icon"></span>
+              Like
+            </button>
+
+            <button class="btn btn-icon">
+              <span><img src="/assets/share.svg" alt="share" class="icon"></span>
+              Share
+            </button>
+
+          </section>
+        </section>
+      </div>
+      <div class="card-container card-style-2">
+        <section class="card-primary-content">
+          <figure class="media">
+            <img src="https://picsum.photos/400/200" alt="img">
+            <figcaption>This is img caption</figcaption>
+          </figure>
+          <header class="card-header">
+            <h2>Title - On Image</h2>
+          </header>
+          <p class="card-body">
+            This hold the summary/ description
+          </p>
+        </section>
+        <section class="actions card-actions">
+          <section class="action-buttons">
+            <button class="btn bg-primary"><a href="/">Action 1</a></button>
+            <button class="btn outline-primary"><a href="/">Action 2</a></button>
+          </section>
+          <section class="action-icons">
+            <button class="btn btn-icon">
+              <span><img src="/assets/like.svg" alt="like" class="icon"></span>
+              Like
+            </button>
+
```

Comment:
HTML files can be formatted using prettier.

Reply:
already did